### PR TITLE
style(htmllint): Tweaking a few items based on HTMLLint results

### PR DESCRIPTION
**Pull request by pdehaan**
There was a lot of noise from the HTMLLint against .mustache files, but these were a few of the errors that jumped out... or just some weird whitespacing issues.

### CODE REVIEW

File: app/scripts/templates/partial/coppa-date-picker.mustache [hunk 1]
```diff
@@ -46,7 +46,7 @@
         </div>
 
         <div class="select-row half-width disabled">
-          <select id="fxa-age-date" disabled="true">
+          <select id="fxa-age-date" disabled="disabled">
```
Comment (pdehaan) [inline app/scripts/templates/partial/coppa-date-picker.mustache]:
```
Error: Bad value true for attribute disabled on element select.

From line 56, column 11; to line 56, column 52
          <select id="fxa-age-date" disabled="true">↩     
```

---

http://www.w3.org/TR/html-markup/select.html

> `disabled = "disabled"` or `""` (empty string) or empty
> Specifies that the element represents a disabled control.

File: app/scripts/templates/marketing_snippet.mustache [hunk 2]
```diff
@@ -1,7 +1,7 @@
   {{#showSignUpMarketing}}
     <aside class="marketing desktop image-snippet default">
       <div class="marketing-spacer">
-        <span class="android-logo" />
+        <span class="android-logo"></span>
```
Comment (pdehaan) [inline app/scripts/templates/marketing_snippet.mustache]:
```
Error: Self-closing syntax (/>) used on a non-void HTML element. Ignoring the slash and treating as a start tag.
From line 9, column 10; to line 9, column 38
↩         <span class="android-logo" />↩     
Error: End tag div seen, but there were open elements.
From line 13, column 8; to line 13, column 13
a>↩       </div>↩↩</bo
Error: Unclosed element span.
From line 9, column 10; to line 9, column 38
↩         <span class="android-logo" />↩     
```

Apparently `<span />` is invalid markup??! Used the slightly more verbose and annoying `<span></span>`.

File: app/scripts/templates/settings.mustache [hunk 3]
```diff
@@ -32,6 +32,7 @@
           <a id="signout" href="#">{{#t}}Sign out{{/t}}</a>
         </li>
       {{/showSignOut}}
+    </ul>
```
Comment (pdehaan) [inline app/scripts/templates/settings.mustache]:
```
app/scripts/templates/settings.mustache [L35:C12] End tag “section” seen, but there were open elements.
app/scripts/templates/settings.mustache [L18:C22] Unclosed element “ul”.
```

Seems legit.

File: app/scripts/templates/marketing_snippet.mustache (hunk 2 above)
Comment (TDA) [inline app/scripts/templates/marketing_snippet.mustache]: Yeah, we are using html as the doctype aren't we? self-closing spans are valid only with xhtml iirc.

File: app/scripts/templates/partial/coppa-date-picker.mustache (hunk 1 above)
Comment (TDA) [inline app/scripts/templates/partial/coppa-date-picker.mustache]: Can't we just leave it empty? Do we really need to do disabled: "disabled"?

File: app/scripts/templates/settings/avatar_gravatar.mustache [hunk 4]
```diff
@@ -6,7 +6,7 @@
   <section>
     <div class="avatar-wrapper with-default">
       {{#gravatar}}
-        <img src="{{ gravatar }}" id="gravatar"/>
+        <img src="{{ gravatar }}" id="gravatar" alt="Avatar" />
```
Comment (TDA) [inline app/scripts/templates/settings/avatar_gravatar.mustache]: nit: alt can be set to "Gravatar" as well. Most users are aware of what that means.

**Comment by TDA** [comment]
Other than those nits, LGTM @pdehaan . CircleCI seems to be having an outage.

File: app/scripts/templates/partial/coppa-date-picker.mustache (hunk 1 above)
Comment (pdehaan) [inline app/scripts/templates/partial/coppa-date-picker.mustache]: Apparently this is also valid, per https://validator.w3.org/nu/:
```html
<select id="fxa-age-date" disabled>
```

File: app/scripts/templates/settings/avatar_gravatar.mustache (hunk 4 above)
Comment (vladikoff) [inline app/scripts/templates/settings/avatar_gravatar.mustache]: yea good nit, will manually merge with this fix.

**Pull request closed without merging**